packages/react-ui-components/package.json

```diff
@@ -51,6 +51,7 @@
 	"dependencies": {
 		"@kelvininc/ui-components": "0.5.3",
 		"@monaco-editor/react": "^4.4.6",
+		"@floating-ui/dom": "^0.4.0",
```

Comment:
why do you need it?

Reply:
I need it for the DEFAULT_DROPDOWN_POSITION_OPTIONS